### ISSUE TITLE
etherpad start fail with abiword

### ISSUE THREAD

**Issue by chicol** [ether/etherpad-lite#3157]
etherpad version: v1.6.1
abiword version : v3.0.2

etherpad always start fail when i enable abiword in setting.json .
`"abiword" : "/usr/local/bin/abiword",`

console err message:
Ensure that all dependencies are up to date...  If this is the first time you have run Etherpad please be patient.
Ensure jQuery is downloaded and up to date...
Clearing minified cache...
Ensure custom css/js files are created...
Started Etherpad...
[2017-04-01 14:33:59.709] [ERROR] console - TypeError: stdoutCallback is not a function
    at ChildProcess.<anonymous> (/opt/etherpad/etherpad-lite/src/node/utils/Abiword.js:94:7)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
[2017-04-01 14:33:59.710] [INFO] console - graceful shutdown...
[2017-04-01 14:33:59.711] [INFO] console - Installed plugins: 
[2017-04-01 14:33:59.714] [INFO] console - Report bugs at https://github.com/ether/etherpad-lite/issues
[2017-04-01 14:33:59.715] [INFO] console - Your Etherpad version is 1.6.1 (ed029b5)
[2017-04-01 14:33:59.831] [INFO] console - You can access your Etherpad instance at http://0.0.0.0:9001/
[2017-04-01 14:33:59.831] [INFO] console - The plugin admin page is at http://0.0.0.0:9001/admin/plugins
[2017-04-01 14:33:59.834] [INFO] console - db sucessfully closed.

ethpad start successful without abiword plugin.

**Comment by chicol** [comment]
@JohnMcLear

**Comment by webzwo0i** [comment]
What OS are you using?

**Comment by webzwo0i** [comment]
sorry, code is a little bit hard to follow there... Can you give https://github.com/ether/etherpad-lite/commit/06046493bf375b70ab1c655aceb928208059b9de a try?
Maybe it's a good idea to not restart abiword if it exists with errors. If it cannot start (because of missing AbiCommand plugin) it should not loop endlessly.

**Comment by chicol** [comment]
@webzwo0i thanks for your reply.
OS: centos 7.3.1611
I cann't have a try with [0604649](https://github.com/ether/etherpad-lite/commit/06046493bf375b70ab1c655aceb928208059b9de) because the environment doesn't exsit now.
I tried install etherpad 3.1 on ubuntu, it's ok.
